packages/nodes-base/nodes/HaloPSA/descriptions/ClientDescription.ts

```diff
@@ -0,0 +1,313 @@
+		name: 'clientName',
+		type: 'string',
+		default: '',
+		required: true,
+		displayOptions: {
+			show: {
+				operation: [
+					'create',
+				],
+				resource: [
+					'client',
+				],
+			},
+		},
+		description: 'Enter client name',
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		default: {},
+		placeholder: 'Add Field',
+		displayOptions: {
+			show: {
+				operation: [
+					'create',
+				],
+				resource: [
+					'client',
+				],
+			},
+		},
+		options: [
+			{
+				displayName: 'Account Status',
+				name: 'inactive',
+				type: 'options',
+				default: false,
+				options: [
+					{
+						name: 'Active',
+						value: false,
+					},
+					{
+						name: 'Inactive',
+						value: true,
+					},
+				],
+			},
+			{
+				displayName: 'Notes',
+				name: 'notes',
+				type: 'string',
+				typeOptions: {
+					alwaysOpenEditWindow: true,
+				},
+				default: '',
+			},
+			{
+				displayName: 'VIP',
+				name: 'is_vip',
+				type: 'boolean',
+				default: false,
+				description: 'Whether the client is VIP or not',
+			},
+			{
+				displayName: 'Website',
+				name: 'website',
+				type: 'string',
+				default: '',
+			},
+		],
+	},
+	/* -------------------------------------------------------------------------- */
+	/*                                client:get                                  */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'Client ID',
+		name: 'clientId',
+		type: 'string',
+		required: true,
+		default: '',
+		displayOptions: {
+			show: {
+				resource: [
+					'client',
+				],
+				operation: [
+					'get',
+					'delete',
+				],
+			},
+		},
+	},
+	/* -------------------------------------------------------------------------- */
+	/*                                client:getAll                               */
+	/* -------------------------------------------------------------------------- */
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				resource: [
+					'client',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		default: false,
+		description: 'Whether to return all results or only up to a given limit',
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		default: 50,
+		displayOptions: {
+			show: {
+				resource: [
+					'client',
+				],
+				operation: [
+					'getAll',
+				],
+				returnAll: [
+					false,
+				],
+			},
+		},
+		typeOptions: {
+			minValue: 1,
+			maxValue: 1000,
+		},
+		description: 'Max number of results to return',
+	},
+	{
+		displayName: 'Filters',
+		name: 'filters',
+		type: 'collection',
+		default: {},
+		placeholder: 'Add Field',
+		displayOptions: {
+			show: {
+				resource: [
+					'client',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		options: [
+			{
+				displayName: 'Include Active',
+				name: 'includeactive',
```

Comment:
Why not in camelCase?

Reply:
Is it about how API uses it?

Reply:
yes, but I'll replace it with options field and change to camelCase